scripts/before_install.sh

```diff
@@ -31,6 +31,7 @@ if (! go run scripts/mingoversion.go 1.11 &>/dev/null); then
   pin github.com/elastic/go-sysinfo v1.3.0
   pin google.golang.org/grpc v1.30.0 https://github.com/grpc/grpc-go
   pin github.com/jinzhu/gorm v1.9.16
+  pin github.com/ugorji/go v1.1.10
```

Comment:
how is this related to the `round` changes?

Reply:
It's not. I was lazy and fixed an unrelated CI failure in the same PR.